.circleci/docker/common/install_base.sh

```diff
@@ -42,6 +42,7 @@ install_ubuntu() {
     libsndfile-dev \
     python \
     python-dev \
+    python-pip \
```

Comment:
There are some other pip installations happening (such as this one: https://github.com/pytorch/pytorch/blob/master/.jenkins/caffe2/test.sh#L171) which we might be able to remove

Reply:
Good point, I'll do this in a separate PR as I see a few potentially outdated/unnecessary pip installs (for Ubuntu Trusty) I could probably clean up as well.

Reply:
Looking back at this, I actually feel like the `python` and `python-dev` installations are unnecessary since they're installed later with `conda` anyway.

https://github.com/pytorch/pytorch/blob/master/.circleci/docker/common/install_conda.sh#L62